### ISSUE TITLE
Show header/footer with a delay

### ISSUE THREAD

**Issue by dasheck0** [lawloretienne/QuickReturn#28]
Hi,

first of all great library ;) I have a scrollview with a header. The header is pushed up when scrolling up. As I scroll down the header gets also scrolled down immediately (hence the name quick return I guess). Is there a way to show the header (or footer) with a delay or only when the scrollview can't be scrolled anymore?

best regards 
dasheck

**Comment by lawloretienne** [comment]
Hey @dasheck0 I'm glad you like the library.  If you take a look at what I do in the Twitter example when you are at the top and start scrolling slowly it doesnt immediately hide the tabs so there is a distance that gets scrolled before the tabs start scrolling off screen.  Is this sort of what you were going for?

**Comment by dasheck0** [comment]
Not exactly. The thing with quick return is that the header gets scrolled in immediately. That is also the purpose of this pattern I guess. I wanted the behaviour of a collapsable toolbar, that you can hide the header to get the maximum amount of space available for the scrollview. And when the user scrolls down the header gets also scrolled in but not immediately but with a delay, so that it does not hide the content of the scrollview. 

Something like this but not with the toolbar but with a header view: https://www.youtube.com/watch?v=6gZ_y7MqEGM

**Comment by lawloretienne** [comment]
Can you please demonstrate what this would look like if you had more than 3 items, lets say 30 items, with an updated video?

**Comment by dasheck0** [comment]
Sure. This is not my video by the way. I just googled the desired behaviour. Anyway I'll see if I find another one and edit this post once I found it.
